R/RtGam.R

```diff
@@ -32,9 +32,17 @@
 #'   time. An increase in this value above the default should be done carefully.
 #'   See [penalty_dim_heuristic()] for more information on `m` and when to
 #'   consider changing the default.
+#' @param backend One of `gam` or `bam`; defaults to `gam`. In general, models
+#'   should be fit with [mgcv::gam()]. If [mgcv::gam()] is too slow,
+#'   [mgcv::bam()] converges more quickly but introduces some additional
+#'   numerical error. Note that the `bam` backend uses the `discrete = TRUE`
+#'   option for an additional speedup. See [mgcv::bam()] for more information.
+#' @param user_supplied_args A list of custom arguments to pass to the model
```

Comment:
as you don't have a ... passing here you could do this with that. Really a stylistic choice.

Reply:
https://github.com/CDCgov/cfa-gam-rt/commit/17f48d47ba089e7728ede4031d80ec6de24385e9